src/components/element-editor/conditional-rendering/conditional-rendering.tsx

```diff
@@ -61,12 +59,12 @@ export const ConditionalRendering = () => {
     };
 
     function getTouchedAndErrorsFields() {
```

Comment:
apologies I think there is no need to check the length too and we can rename the function

```
function checkIsValid() {
        try {
            const validationSchema = schema.pick(["validation"]);

            validationSchema.validateSync({
                validation: validationValues,
                abortEarly: false,
            });
            return false;
        } catch (error) {
            return Yup.ValidationError.isError(error);
        }
    }
```